packages/lib-kaspa/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfns/lib-kaspa",
-  "version": "0.4.3",
+  "version": "0.5.7",
```

Comment:
big jump here

Reply:
I don't think it's been published before 😄